evap/evaluation/tests/test_models.py

```diff
@@ -363,6 +367,27 @@ def test_inactive_users_shown(self):
         self.assertIn(active_user, user_list)
         self.assertIn(inactive_user, user_list)
 
+    def test_can_be_marked_inactive_by_manager(self):
+        user = mommy.make(UserProfile)
+        mommy.make(Evaluation, participants=[user], state="new")
+        self.assertFalse(user.can_be_marked_inactive_by_manager)
+
+        contributor = mommy.make(UserProfile)
+        mommy.make(Contribution, contributor=contributor)
+        self.assertFalse(contributor.can_be_marked_inactive_by_manager)
+
+        user2 = mommy.make(UserProfile, groups=[Group.objects.get(name="Reviewer")])
```

Comment:
I'd name him `reviewer` and make a new user for the other group (`grade_publisher`)